app/src/main.c

```diff
@@ -20,6 +24,17 @@
 int
 main(int argc, char *argv[]) {
 #ifdef __WINDOWS__
+    int wargc;
+    wchar_t **wargv = CommandLineToArgvW(GetCommandLineW(), &wargc);
+
+    if (argc != wargc) {
```

Comment:
Is it possible? Shouldn't it be an assertion?

Reply:
From my point of view, it wouldn't. However, it doesn't indicate it in the official document.